greedy_method/greedy_coin_change.py

```diff
@@ -0,0 +1,48 @@
+"""
+Test cases:
+find_minimum_change(987)
+500 100 100 100 100 50 20 10 50 2
+find_minimum_change(500)
+500
+find_minimum_change(0)
+The total value cannot be zero or negetive
+find_minimum_change(-96)
+The total value cannot be zero or negetive
+find_minimum_change(56)
+50 5 1
+"""
+
+
+def find_minimum_change(V):
+    total_value = int(V)
+    # All denominations of Indian Currency
+    denominations = [1, 2, 5, 10, 20, 50, 100, 500, 2000]
```

Comment:
Maybe you could add an optional argument to the function so that the user can add their own custom denominations.